torch/ao/quantization/qconfig.py

```diff
@@ -325,6 +328,12 @@ def get_default_qat_qconfig(backend='fbgemm', version=1):
                                                                 quant_min=0,
                                                                 quant_max=255),
                               weight=default_per_channel_weight_fake_quant)
+        if backend == 'x86':
+            qconfig = QConfig(activation=FakeQuantize.with_args(observer=MovingAverageMinMaxObserver,
+                                                                quant_min=0,
+                                                                quant_max=255,
+                                                                reduce_range=True),
+                              weight=default_per_channel_weight_fake_quant)
```

Comment:
nit: I feel ideally we want to rename this to include `symmetric` in the name, it's not obvious that this is symmetric quant config

Reply:
It's out of scope of this PR. Maybe do it in another PR.